db/migration/1678783599815-SplitHideTitleAnnotationIntoMultipleFlags.ts

```diff
@@ -0,0 +1,25 @@
+import { MigrationInterface, QueryRunner } from "typeorm"
+
+export class SplitHideTitleAnnotationIntoMultipleFlags1678783599815
+    implements MigrationInterface
+{
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        queryRunner.query(`
+            UPDATE charts
+            SET config = JSON_REPLACE(config, "$.hideTitleAnnotation", JSON_OBJECT("entity", TRUE, "time", TRUE, "change", TRUE))
+            WHERE config->"$.hideTitleAnnotation" IS true
```

Comment:
sadly, `json_extract(...) IS TRUE` doesn't work as expected in mysql 8; it also pull up `false` values (see https://github.com/owid/owid-grapher/pull/1268 and the mysql issue linked there)

the less intuitive but more correct way is:
```suggestion
            WHERE config->>"$.hideTitleAnnotation" = "true"
```

please also make sure to change it below ⬇️